### ISSUE TITLE
How to use masking file

### ISSUE THREAD

**Issue by sekhwal** [tseemann/snippy#498]
Hi,
I am working to identify hqSNPs using snippy. I used Mummer for masking the repeats in the reference genome. I am trying to understand how to use the output file (Mummer-masked file that contains regions) in the following command at snippy or any way to mask the repeats using snippy.

snippy-multi input.tab --ref Reference.gbk --cpus 16 > runme.sh

Thanks!

**Comment by mikeyweigand** [comment]
I had a related issue. The README hints that genomic regions can be masked by passing a bed file to parameter '--mask'. However, this option is not listed in the output of 'snippy --help' and attempting to use it anyway returns error 'Unknown option: mask'. I often resort to just masking portions of my reference fasta with Ns, but this comes at the cost of SNP annotations I'd get by using a genbank reference.

**Comment by Pragungrb** [comment]
I ran into the same issue, did any of you solve the issue? @sekhwal @mikeyweigand

**Comment by sekhwal** [comment]
No! I have not tried masking the reference genome after the conversation.

**Comment by mikeyweigand** [comment]
@Pragungrb I determined that the '--mask' parameter was only applicable when running 'snippy-core' which probably makes sense because avoids potential downstream artefacts during core calculation if masking isn't consistent among the initial snp calculations. But I still think it'd be useful to investigate the initial snippy outputs with annotations AND the ability to mask defined regions.

**Comment by Pragungrb** [comment]
@mikeyweigand Thank you for your reply. It worked for me as well once I tried with snippy-core.
